app/assets/javascripts/range.js

```diff
@@ -0,0 +1,32 @@
+function getVals() {
+    // Get slider values
+    var parent = this.parentNode;
+    var slides = parent.getElementsByTagName("input");
+    var slide1 = parseFloat(slides[0].value);
+    var slide2 = parseFloat(slides[1].value);
+    // Neither slider will clip the other, so make sure we determine which is larger
+    if (slide1 > slide2) {
+      var tmp = slide2;
+      slide2 = slide1;
+      slide1 = tmp;
+    }
+    
+    var labelTargetSelector = parent.dataset['target'];
+    var displayElement = document.querySelector(labelTargetSelector);
+    displayElement.innerHTML = "$" + slide1 + " - $" + slide2;
+  }
+  
+  $( document ).on('turbolinks:load', function() {

```

Comment:
give meaningful names to javascript methods in order to describe what you've done inside it